<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphicsPage</class>
 <widget class="QWidget" name="GraphicsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>504</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_OpenGL">
     <property name="title">
      <string>OpenGL</string>
     </property>
     <layout class="QGridLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="1" column="2">
       <widget class="QComboBox" name="antialiasingSamplesComboBox">
        <item>
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>16</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Anti-aliasing samples:</string>
        </property>
        <property name="buddy">
         <cstring>antialiasingSamplesComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="trilinearFilteringCheckBox">
        <property name="text">
         <string>Trilinear filtering</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="softwareOpenGLCheckBox">
        <property name="text">
         <string>Software rendering</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_Room">
     <property name="title">
      <string>Room</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_1">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="1">
       <widget class="QCheckBox" name="drawNotMappedExits">
        <property name="text">
         <string>Draw unmapped exits</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="drawUpperLayersTextured">
        <property name="text">
         <string>Draw upper layers textured</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="drawDoorNames">
        <property name="text">
         <string>Draw hidden door names</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="updated">
        <property name="text">
         <string>Show outdated room hint</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_Colour">
     <property name="title">
      <string>Colors</string>
     </property>
     <layout class="QGridLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="bgLabel">
        <property name="text">
         <string>Background:</string>
        </property>
        <property name="buddy">
         <cstring>bgChangeColor</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="darkLabel">
        <property name="text">
         <string>Dark w/o Sundeath:</string>
        </property>
        <property name="buddy">
         <cstring>darkPushButton</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="darkLitPushButton">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="darkLitLabel">
        <property name="text">
         <string>Light w/o Sundeath:</string>
        </property>
        <property name="buddy">
         <cstring>darkLitPushButton</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="darkPushButton">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="bgChangeColor">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="connectionNormalLabel">
        <property name="text">
         <string>Normal Connection:</string>
        </property>
        <property name="buddy">
         <cstring>connectionNormalPushButton</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="connectionNormalPushButton">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Modding</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="resourceLineEdit">
        <property name="placeholderText">
         <string>Path to resources</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resourcePushButton">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_Advanced">
     <property name="title">
      <string>Advanced Settings</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>antialiasingSamplesComboBox</tabstop>
  <tabstop>trilinearFilteringCheckBox</tabstop>
  <tabstop>softwareOpenGLCheckBox</tabstop>
  <tabstop>drawUpperLayersTextured</tabstop>
  <tabstop>drawNotMappedExits</tabstop>
  <tabstop>drawDoorNames</tabstop>
  <tabstop>updated</tabstop>
  <tabstop>bgChangeColor</tabstop>
  <tabstop>darkPushButton</tabstop>
  <tabstop>darkLitPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
